libs/vm_modules/include/vm_modules/math/random.hpp

```diff
@@ -28,19 +29,31 @@ namespace vm_modules {
 template <typename T>
 fetch::meta::IfIsInteger<T, T> Rand(fetch::vm::VM *, T const &a = T{0}, T const &b = T{100})
 {
-  std::random_device               rd;
-  std::mt19937_64                  mt(rd());
-  std::uniform_int_distribution<T> dist{a, b};
-  return dist(mt);
+  if (a == b)
```

Comment:
It makes more sense to me to have the following and simply catch the error case:

```suggestion
  if (a >= b)
  {
    vm->RuntimeError("Invalid random range requested");
    return T{0};
  }
```